### DISCUSSION TITLE
my school does not allow emails outside of the domain. is there anyway i can still get it with this limitation

### DISCUSSION THREAD

**Discussion by banana-man10** [community/community#69158] [GitHub Education]
### Hi everyone,

hello.
i have been on and off of github since june about the edu pack. i tell them that i cant get the verification code. is there any way how i can either 1 get the verifcation code or 2 prove to them that i am the owner of the school email. again i have been talking with them since june. ive contacted my schools IT Just for them to basically say fuck you to my face. i have a microsoft account with my school email because it was setup by the school. i dont know what to do and this has been hell

**Comment by samus-aran** [comment]
Hi @Silverstero  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍

**Comment by Ionut4428** [comment]
Hello, did you solve this ? i'm facing the same problem... @banana-man10
